druid/src/widget/slider.rs

```diff
@@ -227,6 +227,14 @@ impl Widget<f64> for Slider {
             ..Default::default()
         }
     }
+
+    fn type_name(&self) -> &'static str {
+        "Slider"
+    }
+
+    fn short_type_name(&self) -> &'static str {
+        "Slider"
+    }
```

Comment:
never seen this in other widgets before :eyes: . The default implementation should be fine

Reply:
You are right, Thanks :)